### PR TITLE
Added support for custom sort object

### PR DESCRIPTION
**Pull request by antonsamper**
This would fix https://github.com/danpaz/bodybuilder/issues/107

### CODE REVIEW

**Comment by antonsamper** [comment]
It's failing on Node v0.x because of `Object.assign`. I'm not sure if this is an issue as Bodybuilder v2 is going to be for Node v4.x+. What do you think @danpaz?

**Comment by ferronrsmith** [comment]
@antonsamper The project uses lodash, you can do `_.assign({}, value)`

File: src/utils.js [hunk 1]
```diff
@@ -61,22 +61,28 @@ export function boolMerge(newObj, currentObj, boolType = 'and') {
...
  */
-export function sortMerge(current, field, direction) {
-  var payload = {[field]: {order: direction.order || direction }}
+export function sortMerge(current, field, value) {
+  let payload
 
-  var idx = _.findIndex(current, function (o) {
-    return o[field] != undefined
+  if (_.isPlainObject(value)) {
+      payload = { [field]: Object.assign({}, value) }
```
Comment (danpaz) [inline src/utils.js]: For consistency with the codebase please keep indents to 2 spaces

**Comment by antonsamper** [comment]
All fixed now

**Comment by danpaz** [comment]
👍 👍

**Pull request merged**